sdk/docs/storage/README.md

```diff
@@ -62,20 +80,19 @@ If you'd like to contribute to this library, please read the [contributing guide
 Azure SDK for Embedded C is licensed under the [MIT][azure_sdk_for_c_license] license.
 
 <!-- LINKS -->
+[azure_pattern_circuit_breaker]: https://docs.microsoft.com/azure/architecture/patterns/circuit-breaker
```

Comment:
I would remove any references to "circuit breaker pattern" from this document as our SDK does not have support for it built in.

Reply:
We don't have full circuit breakers but we do have some aspects with retry.  Also the article is a good read for handling network error patterns